http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/dao/HttpWsmDao.scala

```diff
@@ -15,14 +15,12 @@ import org.broadinstitute.dsde.workbench.leonardo.dao.LandingZoneResourcePurpose
   WORKSPACE_BATCH_SUBNET
 }
 import org.broadinstitute.dsde.workbench.leonardo.dao.WsmDecoders._
-import org.broadinstitute.dsde.workbench.leonardo.dao.WsmEncoders._
 import org.broadinstitute.dsde.workbench.leonardo.db.WsmResourceType
 import org.broadinstitute.dsde.workbench.leonardo.util.AppCreationException
 import org.broadinstitute.dsde.workbench.model.TraceId
 import org.broadinstitute.dsde.workbench.openTelemetry.OpenTelemetryMetrics
 import org.http4s._
 import org.http4s.circe.CirceEntityDecoder._
-import org.http4s.circe.CirceEntityEncoder._
 import org.http4s.client.Client
 import org.http4s.client.dsl.Http4sClientDsl
 import org.http4s.headers.{`Content-Type`, Authorization}
```

Comment:
It would be great to document this file with a Header explaining that this is a legacy DAO that should not be modified other than eventually removing the Landing Zone Logic.

Reply:
I am not seeing a commit with these headers / comments. Are these coming as part of another PR? It could be part of this PR honestly

Reply:
I added them to the interface WsmDao instead of this concrete implementation

Reply:
Sorry could you point me to it? I can't find it 🤔

Reply:
Haha no worries, I see it now, looks good 👍